index.html

```diff
@@ -1255,6 +1239,231 @@ <h3>
+            </dt>
+            <dd>
+              The localized [=string=].
+            </dd>
+            <dt>
+              `lang` (optional)
+            </dt>
+            <dd>
+              A [=language tag=].
+            </dd>
+            <dt>
+              `dir` (optional)
+            </dt>
+            <dd>
+              The [=text-direction=].
+            </dd>
+          </dl>
+          <p>
+            When a plain [=string=] is used, or when the `lang` or `dir`
+            members are omitted, `lang` falls back the [=language tag=] of the
+            [=language map=] key, and `dir` falls back to the [=manifest/dir=]
+            member of the [=manifest=].
+          </p>
+          <aside class="note">
+            <p>
+              The `dir` member must be specified if the direction of a
+              localized string differs from the default direction set in the
+              manifest ([=manifest/dir=] member). Right-to-left text will
+              require specific direction settings if the manifest's default
+              direction is left-to-right, and vice versa.
+            </p>
+            <p>
+              To support multilingual content and ensure proper display and
+              accessibility, it is possible to specify a different [=language
+              tag=] for a [=localized value=]. This is needed for situations
+              where a term or text must be presented in a language different
+              from the user's set language. For example, this helps ensure that
+              an application name is correctly pronounced by assistive
+              technology, even if it is in a foreign language.
+            </p>
+          </aside>
+          <p>
+            To <dfn>process a `*_localized` text member</dfn>, given [=ordered
+            map=] |json:ordered map|, [=ordered map=] |map:ordered map|,
+            [=string=] |member:string|, and [=text-direction=] |manifest
+            direction:string|:
+          </p>
+          <ol class="algorithm">
+            <li>If |member| does not [=map/exist=] in |json|, return.
+            </li>
+            <li>Let |languageMap| be |json|[|member|].
+            </li>
+            <li>If |languageMap| is not an [=ordered map=], return.
+            </li>
+            <li>Let |languageTags:ordered set| be the [=map/keys=] of
+            |languageMap:ordered map|.
+            </li>
+            <li>Set |map|[|member|] to a new [=ordered map=].
+            </li>
+            <li>[=Set/For each=] |languageTag:string| of |languageTags|, run
+            [=process a localized text value=], passing
+            |languageMap|[|languageTag|] as the [=localized value=],
+            |languageTag|, |map|, |member|, and |manifest direction|.
+            </li>
+          </ol>
+          <p>
+            To <dfn>process a localized text value</dfn>, given [=string=] or
+            [=ordered map=] |localizedValue|, [=string=] |languageTag:string|,
```

Comment:
optional, here and below:

```suggestion
            [=ordered map=] |localizedValue|, [=string=] |defaultLanguageTag:string|,
```

Reply:
This is not necessarily the manifest's default language tag but the language map's key. So I think “languageTag” is correct here.

Reply:
Isn't this used if the value doesn't have a 'dir'? Hence it's the default one for parsing this value?

Reply:
@dmurph This value is the language map's key. It is used if the localized value doesn't have a "lang" member (maybe related: https://github.com/w3c/manifest/pull/1101#discussion_r1676783850). I think the wording "default language tag" could be confusing if we also use "default direction" as the "default language tag" is the language map key (not the root/manifest language), but the "default direction" would be the root/manifest direction.

Reply:
Same idea as my other comment - from the perspective this algorithm, this argument is the 'default value' if one isn't specified. totally optional suggestion, you can keep it the same too.